src/main/scala/strawman/collection/mutable/Set.scala

```diff
@@ -69,9 +69,7 @@ trait Set[A]
   }
 }
 
-object Set {
-  def apply[A](xs: A*): Set[A] = ???
-}
+object Set extends IterableFactory.Delegate[Set](HashSet)
```

Comment:
Here I say that the default `mutable.Set` implementation type is `mutable.HashSet`